data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-relevant-transaction-state-creation-v1.0.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet author="R3.Corda" id="ledger-utxo-relevant-transaction-state-creation-postgres-v1.0" dbms="postgresql">
```

Comment:
For reference, where we've had to deal with DB specific types elsewhere, we have used properties.

E.g.: https://github.com/corda/corda-api/blob/5169296760cc10fbdd07ba3fee2a68227dd6adb6/data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml#L6

This would let you do something like
```
<property name="json-type" value="VARCHAR(MAX)" dbms="hsqldb"/>
<property name="json-type" value="JSONP" dbms="postgresql"/>

...

<column name="custom_representation" type="${json-type}">
    <constraints nullable="false"/>
</column>

```